modules/missenaBidAdapter.js

```diff
@@ -7,6 +16,48 @@ const ENDPOINT_URL = 'https://bid.missena.io/';
 const EVENTS_DOMAIN = 'events.missena.io';
 const EVENTS_DOMAIN_DEV = 'events.staging.missena.xyz';
 
+function getSize(sizesArray) {
+  const firstSize = sizesArray[0];
+  if (typeof firstSize !== 'string') return {};
+
+  const [widthStr, heightStr] = firstSize.toUpperCase().split('X');
+  return {
+    width: parseInt(widthStr, 10) || undefined,
+    height: parseInt(heightStr, 10) || undefined,
+  };
+}
+
+/* Get Floor price information */
+function getFloor(bidRequest) {
+  if (!isFn(bidRequest.getFloor)) {
+    return {};
+  }
+  const sizesArray = getSizeArray(bidRequest);
```

Comment:
I don't think this is necessary, the floors module will already pick size from the request when it makes sense. What this does is force using the first size when there are multiple instead of looking for a floor that applies to all sizes. 

Unless that was your intention I recommend removing this. (I am learning now that a lot of other adapters do this, so I suspect you're just following the pattern, which is unfortunately not a good one).

Reply:
To clarify - if you just call `getFloor()`, it will attempt to find reasonable defaults:

 - `currency` defaults to 'USD'
 - `mediaType` defaults to the request's if it contains only one
 - `size` defaults to the request's if it contains only one (which implies it also has only one mediaType)

Reply:
@dgirardi the functions you mentioned have been removed by @ysfbsf  (code original creator). We will rely on our partners to properly configure their bidFloors (the code was there for corner cases). 

Please let us know if we need to change anything else.